redhat-developer/app-services-cli
# refactor(kafka topic): elucidate optional fields in update interactive

### Description

`rhoas kafka topic update` command while running interactively should be more clear regarding which fields can be skipped. Helper texts have been modified to reflect the same.

fixes #487 

### Verification Steps

1. Run `rhoas kafka topic update <topic-name>`
```
rhoas kafka topic update <topic-name>
```
2. The interactive mode should ask for retention period as:
> Retention Period (ms) [optional]:
3. Inputting `q` should show the following text:
> The period of time in milliseconds the broker will retain a partition log before deleting it. Leave blank to skip updating this value.

### Type of change


### Checklist
